src/models/log-v4.js

```diff
@@ -33,10 +39,10 @@ async function displayLogs (params) {
 
   logStream
     .on('open', (event) => {
-      Logger.debug(`stream opened! ${JSON.stringify({ appId, filter, deploymentId })}`);
+      Logger.debug(`Logs stream (open) ${JSON.stringify({ appId, filter, deploymentId })}`);
     })
     .on('error', (event) => {
-      Logger.error(`an error occured: ${event.detail}`);
+      Logger.debug(`Logs stream (error) ${event.error.message}`);
```

Comment:
Maybe we can add a color for `errors` even if it's in debug mode?

Reply:
Nice suggestion yes. Let's put some red on this ;-)